### PR TITLE
Updated translations

### PR DESCRIPTION
**Pull request by nalcalag**
Task/Issue URL: https://app.asana.com/0/414730916066338/1202248365771000/f

### Description

Updated translations in all languages for downloads and automatic fireproof projects

### Steps to test this PR

- Install from this branch
- Change device language to any (but English)
- [ ]  Check downloads section is translated
- [ ] Check Settings > Fireproof Sites screen is translated

### CODE REVIEW

File: app/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -475,11 +476,11 @@
         <item quantity="other"><![CDATA[ intentaban rastrearte aquí. <br/><br/>¡Los he bloqueado!<br/><br/> ☝️Puedes consultar la barra de navegación para ver quién está tratando de rastrearte cuando visitas un nuevo sitio.️]]></item>
     </plurals>
     <string name="daxNonSerpCtaText"><![CDATA[A medida que navegues y te desplaces, bloquearé rastreadores molestos. <br/><br/>Adelante, ¡sigue navegando!]]></string>
-    <string name="daxMainNetworkCtaText">¡Advertencia! No puedo evitar que %1$s vea tu actividad en %2$s &lt;br/&gt;&lt;br/&gt;Pero si navegas conmigo, puedo reducir lo que %3$s sabe de ti en general bloqueando sus rastreadores en muchos otros sitios web.</string>
+    <string name="daxMainNetworkCtaText">¡Advertencia! No puedo evitar que %1$s vea tu actividad en %2$s\n\nPero si navegas conmigo, puedo reducir lo que %3$s sabe de ti en general bloqueando sus rastreadores en muchos otros sitios web.</string>
```
Comment (cmonfortep) [inline app/src/main/res/values-es/strings.xml:479]: Can we check if this change will be an issue?

File: app/src/main/res/values-es/strings.xml [hunk 2]
```diff
@@ -589,7 +599,7 @@
     <!--  Global Privacy Control-->
     <string name="globalPrivacyControlActivityTitle">Control Global de Privacidad (GPC)</string>
     <string name="globalPrivacyControlDescription">
-        <![CDATA[DuckDuckGo bloquea automáticamente muchos rastreadores. Con el Control Global de Privacidad (GPC), también puedes solicitar a los sitios web participantes que restrinjan la venta o el uso compartido de tus datos personales con otras empresas. <u><a href="">Más información</a><u/>]]>
+        <![CDATA[DuckDuckGo bloquea automáticamente muchos rastreadores. Con el Control Global de Privacidad (GPC), también puedes solicitar a los sitios web participantes que restrinjan la venta o el uso compartido de tus datos personales con otras empresas. <u><a href=\"\">Más información</a><u/>]]>
```
Comment (cmonfortep) [inline app/src/main/res/values-es/strings.xml:602]: same here, can we validate if that change doesn't break the link?

**Review by cmonfortep — COMMENTED** [review]
@nalcalag it seems we are bringing more translations that are not related to downloads and automatic fireproof.

For us to test this correctly, could you add to the test cases which screens have been affected? I'm fine if you want to discard copy updates that are not related to your project, whatever works best for you. 

FYI: sometimes translations bring changes that can break the UI ([see this iOS bug for context](https://app.asana.com/0/904401899170/1202047947332591/f)). I've left a comment on the Spanish strings file as an example of a change we need to validate (haven't checked other languages).

File: app/src/main/res/values-es/strings.xml (hunk 2 above)
Comment (nalcalag) [reply app/src/main/res/values-es/strings.xml:602]: 🤯

**Comment by agustincantero** [comment]
In the download section in Spanish in the latest version (5.125.0) It is not translated yet.

**Pull request merged**